pkgs/by-name/li/liquibase/package.nix

```diff
@@ -65,6 +65,7 @@ stdenv.mkDerivation (finalAttrs: {
       cat > $out/bin/liquibase <<EOF
       #!/usr/bin/env bash
       # taken from the executable script in the source
```

Comment:
Nit: Presuming that the analytics part is not "taken from the executable script in the source", could you put your change in front of that comment?

Reply:
Will adjust it, thought the same after commiting 😆